### PR TITLE
Updated open endpoint URLs

### PR DESCRIPTION
**Pull request by naveenaluru**
Updated open endpoint URLs with new domain name and updated metadata self url.

### CODE REVIEW

**Comment by daliboz** [comment]
Will need to hold this update until after deployment of the open endpoint (and validation). We'll also need to announce this upcoming change to all our partners.

Is this going out in today's deployment?

**Comment by naveenaluru** [comment]
yes, this is targeted for today's deployment.

**Comment by daliboz** [comment]
+1 other than holding it until after deployment (I've announced the upcoming change to consumers).

File: content/may2015.md [hunk 1]
```diff
@@ -54,12 +54,12 @@ We encourage you to explicitly request one of these types via the `Accept` heade
...
 
-The FHIR URL takes the form `fhir.sandboxcernerpowerchart.com/may2015/open/:ehr_source_id/:resource`.
+The FHIR URL takes the form `fhir.sandboxcernerpowerchart.com/may2015/:ehr_source_id/:resource`.
 
 ### Open Endpoint
 
-The presence of `/open` in the path allows developers to experiment with the service without requiring
-authentication.
+The `fhir-open.sandboxcernerpowerchart.com` domain allows developers to experiment with the service without requiring
+authentication. Use `fhir.sandboxcernerpowerchart.com` domain to access the authenticated endpoint.
```
Comment (dennispatterson) [inline content/may2015.md]: Small grammatical comment.  Either insert "the" after "Use" (i.e. Use the ... domain) or remove "domain" (i.e. Use ... to access)

**Comment by dennispatterson** [comment]
:+1: other than the minor grammatical comment

**Comment by MaxPhilips** [comment]
+1

**Comment by naveenaluru** [comment]
Addressed comment. Will merge after the deployment is completed and validated

**Comment by naveenaluru** [comment]
Deployment is complete and validated. Merging now.

**Pull request merged**